### PR TITLE
ticket UI Issue

### PR DESCRIPTION
**Pull request by niranjansenthilkumar**
implemented done button in TradeItNumericInputCell class, executes under awakenfromNib function

### CODE REVIEW

File: TradeItIosTicketSDK2/TradeItNumericInputCell.swift [hunk 1]
```diff
@@ -2,31 +2,56 @@ import UIKit
...
         if let initialValue = initialValue {
             self.textField.text = "\(initialValue)"
         } else {
             self.textField.text = nil
         }
     }
-
+    
+    func addDoneButtonOnKeyboard(){
+        let doneToolbar: UIToolbar = UIToolbar(frame: CGRect(x: 0, y: 0, width: 320, height: 50))
```
Comment (jsom) [inline TradeItIosTicketSDK2/TradeItNumericInputCell.swift]: What does it look like on devices with width > 320?

Comment (niranjansenthilkumar) [inline TradeItIosTicketSDK2/TradeItNumericInputCell.swift]: screen shot 2017-06-15 at 3 26 20 pm


I tested it on an iPhone 5, and it seems to look relatively fine

Comment (jsom) [inline TradeItIosTicketSDK2/TradeItNumericInputCell.swift]: iPhone 5 width = 320 though. Can you try on an iPhone 6 or 7?

Comment (niranjansenthilkumar) [inline TradeItIosTicketSDK2/TradeItNumericInputCell.swift]: screen shot 2017-06-15 at 4 07 12 pm

**Comment by guillaumedebavelaere** [comment]
@niranjansenthilkumar 
It would be nice if we can still see the input when the keyboard show up on iphone5.
You may want to see what I did for CIMB to have an idea (it is not perfect there is an extra space): https://github.com/tradingticket/TradeItIosTicketSDK2/pull/172/commits/56e2b61df222fca9d5007839afc0ca060ad2206d

**Pull request merged**